Automattic/jetpack
# VideoPress: Add inspector controls to VideoPress Chapters block

Closes #27195

#### Changes proposed in this Pull Request:

* Adds inspector controls to the VideoPress Chapters block with `videoPressBlockId` and `style` attributes

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Add a VideoPress block, upload a video and give it a title
* Follow the instructions defined here to add a Video Chapters block: https://github.com/Automattic/jetpack/pull/27241
* Click on the block and confirm that the video appears on the list
* Check that the style is set to List as default
* Check that changes are persisted when saving the post

![2022-11-02_18-47-55](https://user-images.githubusercontent.com/8486249/199608939-85459244-a0c8-47a4-9697-08ba89af7b96.png)
